metabase/metabase
# New UI to change the color palette in Admin

Epic https://github.com/metabase/metabase/issues/22816

Adds basic UI to set the extended color palette in Admin. Known issues:
- The palette itself is not changed yet and the same color is used for the entire row in the table.
- There is no chart in the preview section yet.
- To reset a specific color, clear the input for that color.
- Color generation is not supported

How to test:
- `yarn dev-ee`
- Go to Admin -> Appearance
- Make sure it's possible to set all existing colors like it was before.

<img width="1394" alt="Screenshot 2022-05-25 at 17 57 08" src="https://user-images.githubusercontent.com/8542534/170293177-02e9ecf2-7d6c-4208-afb2-d85577ca2826.png">
